### ISSUE TITLE
Model name '/deepke/DeepKE/example/ner/standard/checkpoints/' not found in model shortcut name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese, bert-base-german-cased, bert-large-uncased-whole-word-masking, bert-large-cased-whole-word-masking, bert-large-uncased-whole-word-masking-finetuned-squad, bert-large-cased-whole-word-masking-finetuned-squad, bert-base-cased-finetuned-mrpc). Assuming '/deepke/DeepKE/example/ner/standard/checkpoints/' is a path or url to a directory containing tokenizer files.

### ISSUE THREAD

**Issue by whm233** [zjunlp/DeepKE#332]
**Describe the bug**
> A clear and concise description of what the bug is.

**Environment (please complete the following information):**
 - OS: [e.g. mac / window]
 - Python Version [e.g. 3.6]

**Screenshots**
> If applicable, add screenshots to help explain your problem.

**Additional context**
> Add any other context about the problem here.

**Comment by whm233** [comment]
是不是在下载的预训练文件夹里还要放上预训练模型文件？

**Comment by xxupiano** [comment]
predict时需要

**Comment by zxlzr** [comment]
您好，您没有放预训练文件

**Comment by whm233** [comment]
预训练文件里没有这些模型吧，那五个我都下载了。但是还是跑通了即使报了错，很神奇



发自我的iPhone


------------------ Original ------------------
From: Eric ***@***.***&gt;
Date: Tue,Sep 5,2023 8:53 PM
To: zjunlp/DeepKE ***@***.***&gt;
Cc: whm233 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [zjunlp/DeepKE] Model name'/deepke/DeepKE/example/ner/standard/checkpoints/' not found in model shortcutname list (bert-base-uncased, bert-large-uncased, bert-base-cased,bert-large-cased, bert-base-multilingual-uncased,bert-base-multilingual-cased, bert-base-chinese, bert-base-german-cased,bert-large-uncased-whole-word-masking, bert-large-cased-whole-word-masking,bert-large-uncased-whole-word-masking-finetuned-squad,bert-large-cased-whole-word-masking-finetuned-squad,bert-base-cased-finetuned-mrpc). Assuming'/deepke/DeepKE/example/ner/standard/checkpoints/' is a path or url to adirectory containing tokenizer files. (Issue #332)





 
您好，您没有放预训练文件

**Comment by zxlzr** [comment]
从报错看您可能是预训练文件夹里缺失了文件（看起来像tokenizer files），建议你检查下是否与漏文件

**Comment by zxlzr** [comment]
请问您问题是否已解决

**Comment by whm233** [comment]
还没有去调试



发自我的iPhone


------------------ Original ------------------
From: Eric ***@***.***&gt;
Date: Wed,Sep 6,2023 1:28 PM
To: zjunlp/DeepKE ***@***.***&gt;
Cc: whm233 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [zjunlp/DeepKE] Model name'/deepke/DeepKE/example/ner/standard/checkpoints/' not found in model shortcutname list (bert-base-uncased, bert-large-uncased, bert-base-cased,bert-large-cased, bert-base-multilingual-uncased,bert-base-multilingual-cased, bert-base-chinese, bert-base-german-cased,bert-large-uncased-whole-word-masking, bert-large-cased-whole-word-masking,bert-large-uncased-whole-word-masking-finetuned-squad,bert-large-cased-whole-word-masking-finetuned-squad,bert-base-cased-finetuned-mrpc). Assuming'/deepke/DeepKE/example/ner/standard/checkpoints/' is a path or url to adirectory containing tokenizer files. (Issue #332)





 
请问您问题是否已解决

**Comment by whm233** [comment]
(deepke) root@debian10:/deepke/DeepKE/example/triple# ls /deepke/DeepKE/example/ner/standard/checkpoints
added_tokens.json  eval_results.txt   pytorch_model.bin        tokenizer_config.json
config.json        model_config.json  special_tokens_map.json  vocab.txt    。
您好，这是我的预训练文件，这里面应该是没有遗漏东西的，但是仍然出现了上述的警告。但是最终结果和示例中演戏的结果是一致的，并没有因为警告而出现错误之类的问题出现。









++++

**Comment by xxupiano** [comment]
是warning没报错不影响跑通的话就没问题
